Kros.Utils/src/Kros.Utils/Net/IHttpMessageHandlerFactory.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Net.Http;
+
+namespace Kros.Net
+{
+    /// <summary>
+    /// Factory to create <see cref="HttpMessageHandler"/> used in Kros.Net services.
+    /// </summary>
+    public interface IHttpMessageHandlerFacotry
```

Comment:
We do not need this interface. The factory will be just _factory method_:
`public NetworkChecker(Uri serviceAddress, Func<HttpMessageHandler> messageHandlerFactory)`